server/controller/login_controller.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable linebreak-style */
+import { getSingleUser } from '../helper/userHelper';
+
+const wrongDataStatus = (res, message) => {
+  res.status(400).send({
+    status: 400,
+    message,
+  });
+};
+
+export default function login(req, res) {
+  if (req.body.email === undefined) {
+    res.status(400).send({
+      status: 400,
+      message: 'the email is required, please provide it before proceeding',
+    });
+  } else if (req.body.password === undefined) {
+    wrongDataStatus(res, 'the password is required, please provide it before proceeding');
+  } else if (!req.body.password || !req.body.email) {
+    res.status(401).send({
+      status: 401,
+      message: 'You provided a wrong email or password',
+    });
+  } else {
+    const user = getSingleUser(req.body.email);
```

Comment:
User array destructuring here like 
`const [user] = getSingleUser(req.body.email);`
Then use it like:
`if(user){ ... }`